src/modules/items/ItemHandler.ts

```diff
@@ -49,6 +49,16 @@ export default class ItemHandler {
                 type: NotificationConstants.NotificationOption.danger,
             });
         }
+
+        const partyPokemon = App.game.party.getPokemonByName(this.pokemonSelected());
+        if (partyPokemon.breeding && App.game.challenges.list.realEvolutions.active()) {
+            // If the real evolution challenge is active, we prevent using stones on pokemons in the hatchery to prevent exploits
+            return Notifier.notify({
+                message: 'You can\'t use a stone on a Pokémon if it\'s in the hatchery...',
```

Comment:
```suggestion
                message: 'You can\'t use an evolution item on a Pokémon if it\'s in the hatchery...',
```
I believe we refer to all evolution items as stones internally but there are other kinds of evolution items and this message could be confusing.